controllers/idler/idler_controller.go

```diff
@@ -150,13 +156,15 @@ func (r *Reconciler) ensureIdling(logger logr.Logger, idler *toolchainv1alpha1.I
 					appName = pod.Name
 					appType = "Pod"
 				}
-
-				// By now either a pod has been deleted or scaled to zero by controller, idler Triggered notification should be sent
-				if err := r.createNotification(logger, idler, appName, appType); err != nil {
-					logger.Error(err, "failed to create Notification")
-					if err = r.setStatusIdlerNotificationCreationFailed(idler, err.Error()); err != nil {
-						logger.Error(err, "failed to set status IdlerNotificationCreationFailed")
-					} // not returning error to continue tracking remaining pods
+				// Do not send notification if Pod Not managed by a controller and is in completed state
```

Comment:
```suggestion
				// Send notification if the deleted pod was managed by a controller or was a standalone pod that was not completed eg. If a build pod is in "PodCompleted" status then it was not running so there's no reason to send an idler notification
```
I think adding a bit more context here can help when we look at this code in the future since this is not an obvious case

Reply:
Yes will change